spk/rutorrent/Makefile

```diff
@@ -5,12 +5,12 @@ SPK_ICON = src/rutorrent.png
 DSM_UI_DIR = app
 
 # some dependents require C++11 support
```

Comment:
Perhaps something to add in the requirement.mk file (or what ever the name becomes).
Or manage using GCC_MIN_VERSION... food for thoughts.